### PR TITLE
feat: add Draft Property to newly added blogs

### LINKED ISSUE #623: About the i18n of blogs
**Actual Behaviour**

Currently, our blogs are mixed in English and Chinese. All blogs are displayed whether in English or Chinese. This often brings confusion and inconvenience to users of different languages.

**Expected Behaviour**

When the current environment is in English, only English blogs are displayed, and in the Chinese environment, only Chinese blogs are displayed.

After analysis, we can use the [`draft`](https://docusaurus.io/zh-CN/docs/api/plugins/@docusaurus/plugin-content-blog#markdown-frontmatter) property of the markdown file to solve, It means that the current document is still in draft and will not be displayed in the blog.

So because we default to the English environment, add the `draft` property to all Chinese blogs in the `website/blog` directory.

After that, copy all the blog files to the `website/i18n/zh/docusaurus-plugin-content-blog` directory, and then add the `draft` property to all English blogs.

**Note**: draft will still be displayed in development mode, but after build, it works ~

What do you think？

### PR DESCRIPTION
**Pull request by KishaniKandasamy**
Fixes: #623 

Changes:
added draft property to newly added blogs

### CODE REVIEW

File: website/i18n/zh/docusaurus-plugin-content-blog/2021-09-29-release-apache-apisix-2.10.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+---
+title: "Apache APISIX 2.10.0 正式发布，带来第一个 LTS 版本！"
```
Comment (juzhiyuan) [inline website/i18n/zh/docusaurus-plugin-content-blog/2021-09-29-release-apache-apisix-2.10.md]: Do we need this duplicate post? I see there has one `website/blog/2021-09-29-release-apache-apisix-2.10.md`

Comment (KishaniKandasamy) [reply website/i18n/zh/docusaurus-plugin-content-blog/2021-09-29-release-apache-apisix-2.10.md]: yes We can remove it

**Review by juzhiyuan — COMMENTED** [review]
Hi @KishaniKandasamy, I noticed we also have some posts duplicated, do we have to keep them? e.g the `release-apache-apisix-2.10.md` in Chinese.

**Comment by KishaniKandasamy** [comment]
> Hi @KishaniKandasamy, I noticed we also have some posts duplicated, do we have to keep them? e.g the `release-apache-apisix-2.10.md` in Chinese.

No if both english and chinese translations are done for a blog, then we could remove its duplicate and there is no need to add draft property there.I will check it out and remove them in this PR👍

**Comment by KishaniKandasamy** [comment]
> > Hi @KishaniKandasamy, I noticed we also have some posts duplicated, do we have to keep them? e.g the `release-apache-apisix-2.10.md` in Chinese.
> 
> No if both english and chinese translations are done for a blog, then we could remove its duplicate and there is no need to add draft property there.I will check it out and remove them in this PR👍

Hi @juzhiyuan the duplicates are removed now

**Comment by juzhiyuan** [comment]
Thanks! Need @Baoyuantop @guoqqqi to have a look

**Review by Baoyuantop — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**